game_gg.py

```diff
@@ -0,0 +1,98 @@
+
+
+import random
+n=random.randint(0,10)
+print("This is a guessing game where the user needs to guess a number from 0 to 10 inorder to win.")
```

Comment:
For future:
Checkout multi-line printing. 
https://pythonprogramming.net/multi-line-printing-python-3/

It allows you to print multiple lines by calling print ONCE.